scaaml/stats/print_stats.py

```diff
@@ -80,13 +80,13 @@ def __init__(self, measurements_info: Dict,
                 for trace_name in measurements_info
             } for split in Dataset.SPLITS
         }
-        self._split_trace_group_stats = {
+        self._split_trace_group_stats: Dict[str, Dict[str, Any]] = {
```

Comment:
Can't we narrow the `Any` here a bit?
Seems to a be a `List[STDDEVofSTATofTraces]` maybe